server/http/post_action.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
```

Comment:
not sure why this is here, maybe was thought to be a placeholder for PostActions, bu there are none. remove?